src/actions/API.actions.js

```diff
@@ -447,7 +447,7 @@ export function getSettings() {
       settings.checked = false;
       settings.CountryCode = 'US';
       settings.CountryDialCode = '+1';
```

Comment:
Change above 4 settings, should come from Settings.react.js as they are already set default there in UserPreferences.js